### PR TITLE
Add remaining meta tags to Metadata component

### LINKED ISSUE #2026: Add remaining meta tags to metadata component
~Blocked on #1906~

**Is your feature request related to a problem? Please describe.**
There are some meta tags that need adding and values in the config updating from the original metadata refactor.

We should have 
```
<link rel="apple-touch-icon" sizes="72x72" href={`/path/to/72x72.png`} />
<link rel="apple-touch-icon" sizes="96x96" href={`/path/to/96x96.png`} />
<link rel="apple-touch-icon" sizes="128x128" href={`/path/to/128x128.png`} />
<link rel="apple-touch-icon" sizes="144x144" href={`/path/to/144x144.png`} />
<link rel="apple-touch-icon" sizes="152x152" href={`/path/to/152x152.png`} />
<link rel="apple-touch-icon" sizes="192x192" href={`/path/to/192x192.png`} />
<link rel="apple-touch-icon" sizes="384x384" href={`/path/to/384x384.png`} />
<link rel="apple-touch-icon" sizes="512x512" href={`/path/to/512x512.png`} />
<link rel="icon" type="image/png" href={`/path/to/72x72.png`} sizes="72x72" />
<link rel="icon" type="image/png" href={`/path/to/96x96.png`} sizes="96x96" />
<link rel="icon" type="image/png" href={`/path/to/192x192.png`} sizes="192x192"/>
<meta name="apple-mobile-web-app-title" content={brandName}/>
<meta name="application-name" content={brandName}/>
<meta name="theme-color" content={themeColor}/>
<meta name="msapplication-TileColor" content={themeColor}/>
<meta name="msapplication-TileImage" content={`/path/to/144x144.png`}/>
<meta name="mobile-web-app-capable" content="yes"/>
<link rel="apple-touch-startup-image" href={`/path/to/512x512.png`}/>
```

**Describe the solution you'd like**
Add tags to show as part of our metadata.

**Testing notes**
[Tester to complete]

Dev insight: Update unit tests to check for the 4 tags.

**Additional context**

### PR DESCRIPTION
**Pull request by Bopchy**
Resolves #2026

**Overall change:** _Adds the remaining meta tags specified in #2026 to the Metadata component._

**Code changes:**

- _Add remaining tags to Metadata component._
- _Pass service as prop to Metadata component._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

File: src/app/components/Metadata/index.jsx [hunk 1]
```diff
@@ -43,6 +43,38 @@ const getMetaTags = (metaTags, showArticleTags) => {
...
+};
+
+const getIcons = service => {
+  const sizes = ['72x72', '96x96', '192x192'];
+  return sizes.map(size => (
+    <link
+      rel="icon"
+      type="image/png"
+      sizes={size}
+      href={`https://news.files.bbci.co.uk/include/articles/public/${service}/images/icons/icon-${size}.png`}
```
Comment (dr3) [inline src/app/components/Metadata/index.jsx]: Could this maybe be
```jsx
const iconAssetUrl = (service, size) => `https://news.files.bbci.co.uk/include/articles/public/${service}/images/icons/icon-${size}.png`
```
And use it throughout this file please?

**Review by dr3 — CHANGES_REQUESTED** [review]

File: src/app/components/Metadata/index.jsx [hunk 2]
```diff
@@ -43,6 +43,38 @@ const getMetaTags = (metaTags, showArticleTags) => {
...
+  return sizes.map(size => (
+    <link
+      rel="apple-touch-icon"
+      sizes={size}
+      href={`https://news.files.bbci.co.uk/include/articles/public/${service}/images/icons/icon-${size}.png`}
+    />
+  ));
+};
+
+const getIcons = service => {
```
Comment (ghost) [inline src/app/components/Metadata/index.jsx]:
```suggestion
const getIconLinks = service => {
```

**Review by ghost — DISMISSED** [review]

**Comment by DenisHdz** [comment]
We shouldn't update the `package-lock`, can we reset it please?

File: src/app/components/Metadata/index.jsx (hunk 2 above)
Comment (Bopchy) [reply src/app/components/Metadata/index.jsx]: Switched to using iconAssetUrl()

Comment (DenisHdz) [reply src/app/components/Metadata/index.jsx]: I think that we could still use your first approach `getIconLinks`, but calling the `iconAssetUrl` from inside of it.

Comment (Bopchy) [reply src/app/components/Metadata/index.jsx]: 👍 Changed up my first approach though

File: src/app/components/Metadata/index.jsx [hunk 3]
```diff
@@ -43,6 +43,43 @@ const getMetaTags = (metaTags, showArticleTags) => {
   ));
 };
 
+const iconAssetUrl = (service, size) =>
+  `https://news.files.bbci.co.uk/include/articles/public/${service}/images/icons/icon-${size}.png`;
+
+const getIconLinks = (service, iconType) => {
+  const iconSizes = {
```
Comment (dr3) [inline src/app/components/Metadata/index.jsx]: Can you pass these values from the container please? ideally passing `appleTouchIconSizes` and `iconSizes` props containing the arrays of sizes

**Review by dr3 — CHANGES_REQUESTED** [review]

**Review by dr3 — APPROVED** [review]

**Review by DenisHdz — APPROVED** [review]
Thanks for taking over 👍🏻

**Comment by david-boydell** [comment]
This is ready for merge

**Pull request merged**